test/cpp/interop/server_helper.h

```diff
@@ -43,6 +44,16 @@ namespace testing {
 
 std::shared_ptr<ServerCredentials> CreateInteropServerCredentials();
 
+class InteropContextInspector {
+ public:
+  InteropContextInspector (const ::grpc::ServerContext& context);
```

Comment:
I'll ensure the merge window is short :)

On Wed, Jul 15, 2015 at 2:12 PM David G. Quintas <notifications@github.com>
wrote:

> In test/cpp/interop/server_helper.h
> <https://github.com/grpc/grpc/pull/2413#discussion_r34729917>:
>
> > @@ -43,6 +44,16 @@ namespace testing {
> >
> >  std::shared_ptr<ServerCredentials> CreateInteropServerCredentials();
> >
> > +class InteropContextInspector {
> > + public:
> > +  InteropContextInspector (const ::grpc::ServerContext& context);
>
> Done. Note that msan complains about context_ not being used. It'll be
> used alright in a following PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2413/files#r34729917>.
>